docs/src/pages/components/icons/CreateSvgIcon.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import { green } from '@material-ui/core/colors';
+import { createSvgIcon } from '@material-ui/core/utils';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      '& > svg': {
+        margin: theme.spacing(2),
+      },
+    },
+  }),
+);
+
+const HomeIcon = createSvgIcon(
+  <path d="M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z" />,
+  'Home',
+);
+
+export default function SvgIconsColor() {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      <HomeIcon />
+      <HomeIcon color="primary" />
+      <HomeIcon color="secondary" />
+      <HomeIcon color="action" />
+      <HomeIcon color="disabled" />
+      <HomeIcon style={{ color: green[500] }} />
+    </div>
```

Comment:
How about this simplification?

```suggestion
    <HomeIcon />
```

Reply:
The point was to show that this is a fully baked SvgIcon. We could reduce it to a couple of examples:

```suggestion
    <div className={classes.root}>
      <HomeIcon />
      <HomeIcon color="primary" />
    </div>
```

Reply:
Sounds fair